composer.json

```
@@ -1,15 +1,15 @@
 {
     "name": "demvsystems/werte",
-    "type": "library",
     "description": "Abbildung von allgemeinen Werten",
-    "homepage": "https://github.com/demvsystems/werte",
+    "type": "library",
     "authors": [
         {
             "name": "DEMV-Systems",
             "email": "developer@demv.de",
             "homepage": "https://github.com/demvsystems"
         }
     ],
+    "homepage": "https://github.com/demvsystems/werte",
     "require": {
         "php": "^7.1 || ^8.0",
```

```suggestion
        "php": "^7.2 || ^8.0",
```
Sollten wir noch anpassen wenn wir 7.1 raus schmeißen oder?